### DISCUSSION TITLE
How to capture when the video ends?

### DISCUSSION THREAD

**Discussion by szymonleyk** [igordanchenko/yet-another-react-lightbox#151] [Q&A] [answered]
When video ends, I want to do something. But how to handle it?

**Comment by igordanchenko** [comment]
> When video ends, I want to do something.

Can you provide an example?

**Reply by szymonleyk** [reply]
In my application I can use Video component to display video, but before and after it I can do sth.

Steps:
- input text and click button
- display video
- when video ends, redirect to url (e.g http://test.com), OR display another view

I'm asking, how do I know when video ends?

**Reply by igordanchenko** [reply, marked as answer]
There is currently no public API for this. 

I'm not sure I completely understand your use case, but one way to extend the lightbox functionality is to use custom slides - https://yet-another-react-lightbox.com/examples/custom-slides
